pkg/chunk/gcp/bigtable_index_client.go

```diff
@@ -179,7 +197,10 @@ func (s *storageClientColumnKey) BatchWrite(ctx context.Context, batch chunk.Wri
 			rowKeys = append(rowKeys, rowKey)
 			muts = append(muts, mut)
 		}
-
+		if s.writeLimiter.Allow() == false {
+			s.writeLimiter.WaitN(ctx, len(muts))
```

Comment:
What do we want to rate limit on? Mutations? Keys?

Reply:
Mutations!.